### PR TITLE
Created note listing page UI

### PR DESCRIPTION
**Pull request by anmol-kumar0815**

### CODE REVIEW

**Comment by anmol-kumar0815** [comment]
Fixes #5 
@yedhink  _a
Please review.

File: app/javascript/src/components/Dashboard/Notes/Box.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import Note from "./Note";
+
+const Box = ({ notes }) => (
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Box.jsx:5]: This component doesn't resonate with the content it's trying to render. Try to find a better name that makes sense from the business perspective.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import Footer from "./Footer";
+import Header from "./Header";
+
+const Note = ({ note }) => (
+  <Container key={note.id}>
+    <Header title={note.title} />
+    <Description description={note.description} />
+    <Footer note={note} />
+  </Container>
+);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:14]: https://www.loom.com/share/c7d9165b7169425eb65dbab7eaedde97

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 3]
```diff
@@ -43,62 +44,50 @@ const Notes = () => {
...
+              onClick={() => setShowNewNotePane(true)}
+            />
+          }
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+          }}
+        />
+        {notes.length ? (
+          <Box notes={notes} />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:66]: Don't prematurely abstract.

**Review by yedhink — COMMENTED** [review]
@anmol-kumar0815

**Pull request merged**